### ISSUE TITLE
Tracks are still visible even after deleting and active track

### ISSUE THREAD

**Issue by wolny** [ilastik/ilastik#1757]
After deleting an active track, going back in time one still sees the labeling. It's most likely related to some UI caching...

**Comment by ilastikdev** [comment]
yes, it displays the cached frame and then updates it - same as in other workflows/applets

if you jump to the last frame without ever viewing the intermediate frames, then erase the active track, 
now scroll back in time you will see this is not happening because intermediate frames were never cached

**Comment by wolny** [comment]
Thanks for the comment. Though it would be nice to notify volumina about the change and invalidate the cache => better UX. I'll keep it open for now, and we could discuss what to do with it later.

**Comment by ilastikdev** [comment]
in case we do this it should be done also in any applet that has labels (pixel, auto, classifiers)
